src/main/java/ch/njol/skript/bukkitutil/BiomeMappings.java

```diff
@@ -1,58 +1,58 @@
-/**
- *   This file is part of Skript.
- *
- *  Skript is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
+/*
+ * This file is part of Skript.
  *
- *  Skript is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
  *
- *  You should have received a copy of the GNU General Public License
- *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
  *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
  *
- * Copyright 2011-2017 Peter Güttinger and contributors
+ * Copyright 2011-2018 Peter Güttinger and contributors
  */
 package ch.njol.skript.bukkitutil;
 
-import org.bukkit.block.Biome;
-import org.eclipse.jdt.annotation.Nullable;
-
 import ch.njol.skript.Skript;
 import ch.njol.skript.hooks.biomes.BiomeHook;
 import ch.njol.skript.hooks.biomes.BiomeMapUtil.To19Mapping;
 import ch.njol.skript.util.EnumUtils;
+import org.bukkit.block.Biome;
+import org.eclipse.jdt.annotation.Nullable;
 
 /**
  * 1.8 to 1.9 biome name mappings. 1.9 names make no sense.
  * Should be 1.8 compatible, if it's not there is a bug.
+ *
  * @author bensku
  */
 public abstract class BiomeMappings {
-	
 	private final static EnumUtils<Biome> util = new EnumUtils<>(Biome.class, "biomes");
-	
+
 	private final static boolean mapFor19 = Skript.isRunningMinecraft(1, 9);
-	
-	public static @Nullable Biome parse(final String name) {
+
+	public static @Nullable
+	Biome parse(final String name) {
```

Comment:
Two lines? Umm, does not seem like a good idea.

Reply:
My fault, sorry. I don't think so either

Reply:
Actually it's the fault of how it was written that IntelliJ chose to place it this way, the @Nullable in every other place is in the line above the `modifiers ReturnType name(args) {`

Reply:
Oh, that explains a lot. I wonder what I was thinking when I wrote the annotation that way.